react/features/mobile/call-integration/middleware.js

```diff
@@ -261,8 +262,14 @@ function _conferenceWillJoin({ dispatch, getState }, next, action) {
                 });
                 dispatch(appNavigate(undefined));
 
-                // FIXME show a dialog saying "Unable to place the call,
-                // please end any ongoing calls and try again."
+                Alert.alert(
```

Comment:
@saghul What do you think about using the Alert ? Is that okay or should I go with a dialog from base/dialog ?
Downside of Alert is that there's no ad-hoc translate function and the downside of base/dialog is there's no AlertDialog yet and I'd have to create a separate component for that.

Reply:
As discussed offline: we don't love it, but this is an edge case.